tests/foreman/cli/test_contentview.py

```diff
@@ -5323,7 +5322,56 @@ def test_positive_arbitrary_file_repo_promotion(self):
         :expectedresults: Check arbitrary files from FR is available on
             environment
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :CaseLevel: Integration
         """
+        org = make_org({u'name': gen_alphanumeric()})
+        cv = make_content_view({u'organization-id': org['id']})
+        product_info = make_product({u'organization-id': org['id']})
+        repos = [self._make_file_repository_upload_contents(
+            {
+                u'content-type': 'file',
+                u'organization-id': org['id'],
+                u'product-id': product_info['id']
+            }
+        ) for _ in range(2)
```

Comment:
as per stubbed test looks using one repo however here creating two repo ?

Reply:
Right, The thoughts behind this were to test the repo existence in content-view having multiple published repo.
Now I updated the code as per test steps.